src/node.rs

```diff
@@ -21,20 +21,26 @@ use data::{Data, DataIdentifier};
 use error::{InterfaceError, RoutingError};
 use event::Event;
 use id::FullId;
+#[cfg(feature = "use-mock-crust")]
+use id::PublicId;
 #[cfg(not(feature = "use-mock-crust"))]
 use maidsafe_utilities::thread;
 use messages::{CLIENT_GET_PRIORITY, DEFAULT_PRIORITY, RELOCATE_PRIORITY, Request, Response,
                UserMessage};
-use routing_table::Authority;
 #[cfg(feature = "use-mock-crust")]
-use routing_table::RoutingTable;
+use routing_table::{Prefix, RoutingTable};
+use routing_table::Authority;
```

Comment:
I don't think so, actually - note that the previous line is tagged with `#[cfg]`. The compiler doesn't complain about unused imports, so I guess we also use `Authority` outside of `use-mock-crust`.